test/test-applications/integrations/TestApplication.CustomSdk/TestApplication.CustomSdk.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" VersionOverride="1.4.0-rc.2" />
```

Comment:
As discussed privately, please avoid VersionOverride. In main folder there is a *Package.props file where versions can be diefined.

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2069/commits/d62dbc5eedcab7bca73780e970c8e3a63fe6c020